hw2/03-count.js

```diff
@@ -1,3 +1,20 @@
 // Add your code here
+const input = document.querySelector('input');
+const originalText = `It was the best of times, it was the worst of times, it was the age of
+wisdom, it was the age of foolishness, it was the epoch of belief, it
+was the epoch of incredulity, it was the season of Light, it was the
+season of Darkness, it was the spring of hope, it was the winter of
+despair, we had everything before us, we had nothing before us, we were
+all going direct to Heaven, we were all going direct the other way—in
+short, the period was so far like the present period, that some of its
+noisiest authorities insisted on its being received, for good or for
+evil, in the superlative degree of comparison only.`;
 
-input.addEventListener('keydown', handleKeyDown);
+function handleKeyDown() {
+  const text = originalText;
+  const searchInput = input.value;
+  document.getElementById('searchedText').innerHTML = 
+  text.replaceAll(searchInput, `<span style= "background-color: yellow;">${searchInput}</span>`);
+}
+
+input.addEventListener('input', handleKeyDown);
```

Comment:
If the input event is being used then why is the function called handleKeyDown? Perhaps consider renaming this to avoid any confusion.